kubernetes/irs-publisher/templates/deployment.yaml

```diff
@@ -36,6 +36,26 @@ spec:
               value: .Values.kafka.irsGroup
             - name: KAFKA_SIGN_GROUP
               value: .Values.kafka.signGroup
+            - name: CASSANDRA_JOURNAL_KEYSPACE
+              valueFrom:
+                configMapKeyRef:
+                  name: cassandra-configmap
+                  key: cassandra-keyspace
+            - name: CASSANDRA_JOURNAL_KEYSPACE
```

Comment:
I believe that `CASSANDRA_JOURNAL_KEYSPACE` and `CASSANDRA_SNAPSHOT_KEYSPACE` are accidentally repeated in this config.